eng/test/test-cases/SanityCheck/CommandExecutor/CommandExecutorCatchWronglyInApplication.yaml

```diff
@@ -4,12 +4,12 @@ description:
   condition: >-
     CommandExecutor initialized with a topic namespace that is invalid.
   expect: >-
-    Erroneous test case expects CommandExecutor to throw exception incorrectly indicating error in application.
+    Erroneous test case expects CommandExecutor to throw exception incorrectly indicating error is remote.
 
 prologue:
   executors:
   - topic-namespace: "invalid/{modelId}"
   catch:
     error-kind: invalid configuration
```

Comment:
I don't understand the changes to the sanity checks.  There should already be cases to ensure that the IsRemote field is being checked by the METL infrastructure; these cases are to check that the InApplication field is being checked (per the titles of the test cases).

Reply:
ah, in that case I can remove these tests entirely - in-application isn't a field that metl can check for anymore, so these tests would always pass (when they should fail)

Reply:
nope, all languages! It got removed from METL (and the SDKs) here: https://github.com/Azure/iot-operations-sdks/pull/592/files, but the sanity tests didn't get updated

Reply:
this changed quite a bit more than I was expecting - please let me know if this is what you would have expected to happen or not!

Reply:
@jrdouceur, I'm going to merge this PR since it's just into our feature branch, but I'll follow up with you on an email thread to make sure this resolves how we expect before this would go to main!